src/Frontend/Document.php

```diff
@@ -180,6 +195,10 @@ protected function makeView(): View
      */
     protected function makeTitle(): string
     {
+        if (isset($this->premadeData['title'])) {
+            return $this->premadeData['title'];
+        }
+
         $onHomePage = rtrim($this->request->getUri()->getPath(), '/') === '';
 
         return ($this->title && ! $onHomePage ? $this->title.' - ' : '').Arr::get($this->forumApiDocument, 'data.attributes.title');
```

Comment:
Where did this code block go?

Reply:
it still executes a first time when called from the `prepare` method

Reply:
On mobile those lines were marked as removed ..